Makefile

```diff
@@ -246,3 +246,13 @@ update_codegen:
 
 # Dummy force target for pre-build steps
 .PHONY: .FORCE
+
+#currently only for stm32
+#will need different bash window to run the python
+#to send CAN message write
+#can_message.send_message(1) to send can message
```

Comment:
not really sure if this is the best place for this documentation

Reply:
Ah hm. I could put it in the python script or the c project.

Reply:
I'd probably suggest both, and at the top of this file as well